services/harmony/fixtures/uat.urs.earthdata.nasa.gov-443/log-viewer-eve-groups

```diff
@@ -23,4 +23,4 @@ X-Runtime: 0.011620
 Strict-Transport-Security: max-age=31536000
 Transfer-Encoding: chunked
 
-{"user_groups":[{"group_id":"0cf2a427-96cc-453e-ae44-e28dd0958738","name":"Harmony Log Viewers","tag":null,"shared_user_group":false,"created_by":"harmony_uat","app_uid":"harmony_uat","client_id":"abcd"}]}
```

Comment:
I would recommend adding at least one new user (and fixture). We should have one that is only service-deployer and one that is log-viewer-and-service-deployer.

The name of the fixture should explain exactly what it is - for example `log-viewer-eve-groups` - Eve should not also be in the service-deployers group - if she is then we should name the file `log-viewer-and-service-deployer-eve-groups`